Website/src/rendering/src/components/Navigation/AccountPopup.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import useOcUser from '../../hooks/useOcUser';
+import useOcAuth from '../../hooks/useOcAuth';
+import {
+  clearAuthenticationTokens,
+  isAuthenticationEnabled,
+  getLoginUrl,
+  logoutUrl,
+} from '../../services/AuthenticationService';
+
+const AccountPopup = (): JSX.Element => {
+  const router = useRouter();
+  const { user } = useOcUser();
+  const { isAnonymous, isAuthenticated } = useOcAuth();
```

Comment:
After login, the account popup still displays the Login and Register buttons when I tested. Is this normal?

Reply:
Only happens when we are not redirecting on the home page after login

Reply:
Created a bug for that https://dev.azure.com/sitecoredst/Demo/_workitems/edit/7549